cli/lib/kontena_cli.rb

```diff
@@ -6,7 +6,7 @@ module Kontena
   #
   # @param [String,Array<String>] command_line
   # @return [Fixnum] exit_code
```

Comment:
The `@param`, `@raise` and `@return` docs need updating.

Seems that it returns the `Kontena::Command.run` per default in the success case, which is either whatever `execute` returns, or `true` if it does `exit(0)`.